Cargo.toml

```diff
@@ -81,3 +83,13 @@ required-features = ["perms"]
 name = "tx_cache"
 path = "examples/tx_cache.rs"
 required-features = ["perms"]
+
+
+# [patch.crates-io]
+# signet-bundle = { path = "../sdk/crates/bundle"}
+# signet-constants = { path = "../sdk/crates/constants"}
+# signet-evm = { path = "../sdk/crates/evm"}
+# signet-extract = { path = "../sdk/crates/extract"}
+# signet-tx-cache = { path = "../sdk/crates/tx-cache"}
+# signet-types = { path = "../sdk/crates/types"}
+# signet-zenith = { path = "../sdk/crates/zenith"}
```

Comment:
do we want to remove this?

Reply:
i'd like to keep, as i use them regularly